extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -34,6 +42,12 @@ class AmpSocialShare extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    this.isExperimentOn_ = isExperimentOn(this.getWin(), EXPERIMENT);
```

Comment:
Is experiment already defined in the `tools/experiments.js`? Also what about docs?